### PR TITLE
TVAULT-4710 create and load nbd driver

### PR DESCRIPTION
**Pull request by trilio-shiwanand**
Load nbd module before and after reboot of compute node(s)

### CODE REVIEW

**Review by abhijeetpatra — APPROVED** [review]

File: ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml [hunk 1]
```diff
@@ -33,3 +58,9 @@
         - python3-tvault-contego
      state: latest
   when: PYTHON_VERSION=="python3"
+
+- name: Remove recently installed openstack-{{OPENSTACK_DIST}} repo
```
Comment (shyam-biradar) [inline ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml:62]: Is this change intentional?

File: ansible/roles/ansible-tvault-contego-extension/tasks/install_ubuntu.yml [hunk 2]
```diff
@@ -10,13 +10,32 @@
...
+    state: "present"
+
 - name: Download contego virtenv deb package
   shell: |
           curl -Og6 http://{{ IP_ADDRESS }}:{{ PORT_NO }}/deb-repo/deb-repo/tvault-contego-extension_{{ TVAULT_PACKAGE_VERSION }}_all.deb
           dpkg --configure -a && apt-get -o Dpkg::Options::="--force-confold" install ./tvault-contego-extension_{{TVAULT_PACKAGE_VERSION}}_all.deb -y
           rm -rf ./tvault-contego-extension_{{ TVAULT_PACKAGE_VERSION }}_all.deb
 
-- debug : msg="curl -Og6 http://{{IP_ADDRESS}}:{{ PORT_NO }}/deb-repo/deb-repo/tvault-contego-extension_{{TVAULT_PACKAGE_VERSION}}_all.deb" verbosity="{{ verbosity_level }}"
+- debug : msg="curl -Og6 http://{{IP_ADDRESS}}:{{ PORT_NO }}/deb-repo/deb-repo/tvault-contego-extension_{{TVAULT_PACKAGE_VERSION}}_all.deb" verbosity={{ verbosity_level }}
```
Comment (shyam-biradar) [inline ansible/roles/ansible-tvault-contego-extension/tasks/install_ubuntu.yml:38]: Is this change intentional?

**Review by shyam-biradar — COMMENTED** [review]
Code looks good. Please check my comments.

File: ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml (hunk 1 above)
Comment (trilio-shiwanand) [reply ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml:62]: Actually, it's fixed into the **dev-stable/4.2 branch**, this change is yet needed to merge into the **Master branch**

**Review by shyam-biradar — APPROVED** [review]
